src/service/viewport-impl.js

```diff
@@ -964,11 +964,15 @@ export class ViewportBindingNatural_ {
 export class ViewportBindingNaturalIosEmbed_ {
   /**
    * @param {!Window} win
+   * @param {!./ampdoc-impl.AmpDoc} ampdoc
    */
-  constructor(win) {
+  constructor(win, ampdoc) {
```

Comment:
I wanted to highlight that `win` is the main argument. `ampdoc` is a less important thing thing here for now. `ampdoc` is only used to fix an iOS bug. So as soon as that code gets cleaned up, I will remove it.